prio/config.c

```diff
@@ -22,6 +22,12 @@ PrioConfig_maxDataFields(void)
   return (n_roots >> 1) - 1;
 }
 
+int
+PrioConfig_maxIntEntries(int prec)
```

Comment:
Since the input here is a signed `int`, do we want to check whether `prec <= 0` and do something sensible if so?

Reply:
You're right. That check was in encode.c of the old PR and I forgot to port it over. There I just did P_CHECKCB(), but if you point me to an example of error handling, I can add that there.

Reply:
Since this function returns an `int` instead of an error code, you could just return `0` if `prec <= 0`.